frontend/simple/js/state.js

```diff
@@ -134,7 +189,9 @@ const actions = {
         return console.error(`bad action ${type} on ${contractType} (${contractId}):`, entry)
       }
       // TODO: verify each entry is signed by a group member
-      await db.addLogEntry(contractId, entry)
+      let hash = await db.addLogEntry(contractId, entry)
+      // If this is a duplicate the hash will be bogus and we should exit
+      if (!hash) { return }
```

Comment:
According to eslint it's not valid. Actually I don't think there is a valid scenario for this to occur now that we remove the initial event from the synchronize. Provided we can't get duplicate events over the network. In future scenarios of the apps network topology it may be needed.